examples/mockups/product.html

```diff
@@ -59,43 +59,29 @@ <h1 class="ui header massive"><<i>missing</i>> Infographic</h1>
               <h3 class="ui header inverted"><<i>missing</i>> Illustration</h3>
               <h3 class="ui header inverted">Docs as Code</h3>
               <p>
-                Read the Docs simplifies the entire process of building and deploying developer documentation. With support for Sphinx and Mkdocs, you can integrate your code and user-facing documentation using the same tools. Create beautiful documentation easily with themes, and preview every commit with Pull Request previews.
+                Read the Docs simplifies the entire process of building and deploying developer documentation. With support for Sphinx, Mkdocs, and Jupyter Book, you can integrate your code and user-facing documentation using the same tools. Create beautiful documentation easily with themes, and preview every commit with Pull Request previews.
               </p>
               
               <div class="ui list inverted relaxed divided">
-
-                <div class="item">
-                  <i class="icon check circle green"></i>
-                  <div class="content">
-                    <div class="description">Issue Tracking</div>
-                  </div>
-                </div>
-                
-                <div class="item">
-                  <i class="icon check circle green"></i>
-                  <div class="content">
-                    <div class="description">Version Control</div>
-                  </div>
-                </div>
                 
                 <div class="item">
                   <i class="icon check circle green"></i>
                   <div class="content">
-                    <div class="description">Plain Text Markup</div>
+                    <div class="description">Write your docs in plain text for easy review and translation</div>
                   </div>
                 </div>
-                
+
                 <div class="item">
                   <i class="icon check circle green"></i>
                   <div class="content">
-                    <div class="description">Code Reviews</div>
+                    <div class="description">Preview documentation changes before they are merged</div>
                   </div>
                 </div>
                 
                 <div class="item">
                   <i class="icon check circle green"></i>
                   <div class="content">
-                    <div class="description">Automated Tests</div>
+                    <div class="description">Send webhooks to external services when your docs are built</div>
```

Comment:
This is a pretty minor feature that many people don't use... I'd probably focus on something else here.

Reply:
If we want to mention this feature, I'd use say something like "Slack notifications on build failures" which is something that everybody understands and it's easier to sell to non-technical users.

Reply:
Tweaked this sentence a bit